.github/workflows/be-dev-deploy-closed-prs.yml

```
@@ -1,11 +1,20 @@
 name: BE Dev Deploy Closed Prs
 
+#on:
+  #  push:
+  #    branches:
+  #      - be/dev
+  #    paths:
+  #      - 'server/**'
+
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)